src/lib.rs

```diff
@@ -661,13 +669,13 @@ impl Spell {
         let mut instructions_iter = instructions.iter();
         let mut section: Option<u64> = None;
         while let Some(&bits) = instructions_iter.next() {
-            if section.is_some_and(|x| x == METADATA_SECTION) && !(READY_SECTION..=599).contains(&bits)  { // ignore all checks in metadata section
+            if section.is_some_and(|x| x == ABOUT_SECTION) && !(WHEN_CREATED_SECTION..=ABOUT_SECTION).contains(&bits)  {
```

Comment:
### Word of caution
The refactoring of `599` to `ABOUT_SECTION` has potential to cause bugs when new sections are added in the future
This is applicable to all matches of sections

#### Possible remedy
creation of a new constant
> `pub const SECTIONS = 500..=599;`

and accompanying match pattern syntax
> ...
> `section if SECTIONS.constains(&section) => ...`
> ...